examples/gradle/dokka-customFormat-example/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-bin.zip
```

Reply:
Thanks - it actually needs to be updated in a few places!

Reply:
Yeah, I'll update them after Dokka 1.8.10 released.